libpod/boltdb_state.go

```diff
@@ -359,27 +359,24 @@ func (s *BoltState) Container(id string) (*Container, error) {
 	return ctr, nil
 }
 
-// LookupContainer retrieves a container from the state by full or unique
+// LookupContainerID retrieves a container ID from the state by full or unique
 // partial ID or name
-func (s *BoltState) LookupContainer(idOrName string) (*Container, error) {
+func (s *BoltState) LookupContainerID(idOrName string) ([]byte, error) {
```

Comment:
id's are usually strings in the rest of the library, why do []byte here?

Reply:
There's no strong reason for preferring `[]byte` over `string`, I just thought the former would make more sense to distinguish an ID passed from CLI from an ID retrieved from the DB.

Reply:
I think we should probably prefer string here